### DISCUSSION TITLE
Steam servers not connecting

### DISCUSSION THREAD

**Discussion by CodMW2RandomDude** [WilliamVenner/gmpublisher#120] [Help]
i tried to Close steam with the task Manager it didn't work 
even i did Restart Pc
its still didn't work with it
maybe there is way to fix it
its says "Disconnected From Steam Servers."

**Comment by VladyslavIzhak** [comment]
Same issue here

**Comment by ghost** [comment]
You may have a limited account

https://help.steampowered.com/en/faqs/view/71D3-35C2-AD96-AA3A

**Reply by JaydenUB** [reply]
I don't have limited account and the issue is still happening to me
